### DISCUSSION TITLE
Any docker image for building

### DISCUSSION THREAD

**Discussion by dimitrisfinas** [open-telemetry/opentelemetry-cpp#1229] [Q&A] [answered]
hello, do you have any docker image that would contain all prerequisites/third-parties so that we can build a new version more easily?

**Comment by esigo** [comment, marked as answer]
Hi @dimitrisfinas, thanks for reaching out to us.
You may use the [Dockerfile](https://github.com/open-telemetry/opentelemetry-cpp/blob/63f39a5f13a0cadc253f8310e3773f6f0b22f841/docker/ubuntuLatest/Dockerfile) from my PR #1165:
`docker build --build-arg BASE_IMAGE=gcc:8.5 -f ../docker/ubuntuLatest/Dockerfile . -t otel-cpp`.

or the Dockerfile from [main](https://github.com/open-telemetry/opentelemetry-cpp/tree/main/docker/ubuntuLatest).

**Reply by dimitrisfinas** [reply]
That's great ! Thank you, after days fighting to get a working build configuration, this image solved my issue! I will now be able to continue my tests of opentelemetry on C++
